spec/models/book_spec.rb

```diff
@@ -32,15 +32,14 @@
   it { is_expected.to have_and_belong_to_many(:notes) }
 
   describe '.grab_isbns' do
-    before { Book.grab_isbns('Body text (0804720991, 9780804720991) and more text.') }
     it 'adds dirty books from an isbn inside a block of text' do
-      expect(Book.where(isbn_10: '0804720991', dirty: true).exists?).to be_truthy
-      expect(Book.where(isbn_13: '9780804720991', dirty: true).exists?).to be_truthy
+      expect(SyncBookJob).to receive(:perform_later).twice
+      Book.grab_isbns('Body text (0804720991, 9780804720991) and more text.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.